<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Save</class>
 <widget class="QDialog" name="Save">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>315</width>
     <height>156</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	background-color: rgb(248, 248, 255);
	color: rgb(248, 248, 255);
    font: 14pt &quot;黑体&quot;;
    margin-top: 0ex;
    border-width:3px;  
    border-style:solid;
    border-color: rgb(51, 153, 255);  
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/35_all/67.bmp);
background:transparent;
font: 12pt &quot;黑体&quot;;
color: rgb(248, 248, 255);</string>
    </property>
    <property name="text">
     <string>是</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/35_all/67.bmp);
background:transparent;
font: 12pt &quot;黑体&quot;;
color: rgb(248, 248, 255);</string>
    </property>
    <property name="text">
     <string>否</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>25</y>
      <width>65</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/warn.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>47</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>  保存参数设置？</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>82</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>315</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 153, 255); 
font: 12pt &quot;黑体&quot;;
color: rgb(248, 248, 255);</string>
   </property>
   <property name="text">
    <string>  Dat@2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>0</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/35_all/cha.bmp);
background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
